sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md

```diff
@@ -1,14 +1,13 @@
 # Release History
 
-## 1.5.0-beta.3 (Unreleased)
+## 1.5.0-beta.3 (2023-10-19)
```

Comment:
Hmmm, I think we might not need to upgrade the beta version but only upgrade the stable version which means it will be a patch version for AppConfig.